nixpkgs-overlays/thunk-import/default.nix

```diff
@@ -46,7 +44,19 @@ self:
         name = baseNameOf p;
         outPath = self.filterGit p;
       };
+  this = {
+    filterGit = builtins.filterSource (path: type: !(builtins.any (x: x == baseNameOf path) [".git" "tags" "TAGS" "dist"]));
+
+    # Retrieve source that is controlled by the hack-* scripts; it may be either a stub or a checked-out git repo
+    thunkImport = thunkImport;
```

Comment:
You can write `inherit thunkImport;` to do this. It's a bit more idiomatic.

Reply:
Fixed